### ISSUE TITLE
`pos_label` is missing from AUROC

### ISSUE THREAD

**Issue by zhiqiangdon** [Lightning-AI/metrics#1376]
## 🐛 Bug

The `pos_label` argument is missing from AUROC/BinaryAUROC after the recent cleanup: https://github.com/Lightning-AI/metrics/pull/1252

I guess this is a bug, but would like confirm with you.

### To Reproduce

Steps to reproduce the behavior...

### Environment

- TorchMetrics version (and how you installed TM, e.g. `conda`, `pip`, build from source):
- Python & PyTorch Version (e.g., 1.0):
- Any other relevant information such as OS (e.g., Linux):

**Comment by SkafteNicki** [comment]
Hi @zhiqiangdon, thanks for reporting this issue.
In this case it is not an error but instead an explicit design decision. We standardized all binary classification metrics to always require the target to be encoded as [0,1] where the positive class will then be the 1 class.
If you want the 0 class to be the positive class then feed targets as
```python
BinaryAUROC()(preds, 1-target)
```
Sorry for the confusion. I created PR #1383 that will explicit mention that 1 is always considered the positive class.
